flow360/component/simulation/translator/solver_translator.py

```diff
@@ -501,18 +503,55 @@ def actuator_disk_translator(model: ActuatorDisk):
     }
 
 
+def heat_transfer_entity_info_serializer(volume, solid_zone_boundaries: set):
+    """Heat transfer entity info serializer"""
+    for boundary in volume.private_attribute_zone_boundary_names.items:
```

Comment:
Let's raise `Flow360TranslationError` here if the boundary names are not set. You can search how to raise this exception.